tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements.txt
+    pyftpdlib
+    pytest-xdist
+commands =
+    py.test {posargs:tests}
```

Comment:
`-n 4` tells pytest-xdist to run the tests in parallel using 4 CPU cores.  This speeds up the test run from 60 seconds to 40 seconds on my laptop.  It may be useful to preserve this.

`--resultlog` is a deprecated py.test command-line argument that's going to be removed in py.test 4.0.  I think we can drop it.